README.md

```diff
@@ -38,765 +56,196 @@ Given that Webpacker already provides React integration, why would you use "Reac
-  + **raise_on_prerender_error:** Default is false. True will throw an error on the server side rendering. Your controller will have to handle the error.
-
-  Note: client hydration will not trigger for components rendered through XHR. You will have to handle it with javascript.
-  For an example, see [spec/dummy/app/views/pages/xhr_refresh.rb](https://github.com/shakacode/react_on_rails/tree/master/spec/dummy/app/views/pages/xhr_refresh.rb). 
-
-### react_component_hash
-`react_component_hash` is used to return multiple HTML strings for server rendering, such as for
-adding meta-tags to a page. It is exactly like react_component except for the following:
-
-1. `prerender: true` is automatically added to options, as this method doesn't make sense for 
-  client only rendering.
-2. Your JavaScript for server rendering must return an Object for the key `server_rendered_html`.
-3. Your view code must expect an object and not a string.
+1. Edit `app/views/hello_world/index.html.erb` and set `prerender` to `true`.
+2. Refresh the page.
 
-Here is an example of ERB view code:
+This is the line where you turn server rendering on by setting prerender to true:
 
 ```erb
-  <% react_helmet_app = react_component_hash("ReactHelmetApp", prerender: true,
-                                             props: { helloWorldData: { name: "Mr. Server Side Rendering"}},
-                                             id: "react-helmet-0", trace: true) %>
-  <% content_for :title do %>
-    <%= react_helmet_app['title'] %>
-  <% end %>
-  <%= react_helmet_app["componentHtml"] %>
-```
-
-And here is the JavaScript code:
-
-```js
-export default (props, _railsContext) => {
-  const componentHtml = renderToString(<ReactHelmet {...props} />);
-  const helmet = Helmet.renderStatic();
-
-  const renderedHtml = {
-    componentHtml,
-    title: helmet.title.toString(),
-  };
-  return { renderedHtml };
-};
-
+<%= react_component("HelloWorld", props: @hello_world_props, prerender: false) %>
 ```
-  
-### redux_store
-#### Controller Extension
-Include the module `ReactOnRails::Controller` in your controller, probably in ApplicationController. This will provide the following controller method, which you can call in your controller actions:
-
-`redux_store(store_name, props: {})`
 
-+ **store_name:** A name for the store. You'll refer to this name in 2 places in your JavaScript:
-  1. You'll call `ReactOnRails.registerStore({storeName})` in the same place that you register your components.
-  2. In your component definition, you'll call `ReactOnRails.getStore('storeName')` to get the hydrated Redux store to attach to your components.
-+ **props:**  Named parameter `props`. ReactOnRails takes care of setting up the hydration of your store with props from the view.
+For more information on view helpers check our [View Helpers API](docs/api/view-helpers-api.md) and the [Redux Store API](docs/api/redux-store-api.md)  
```

Comment:
We should remove [trailing spaces](https://github.com/markdownlint/markdownlint/blob/e3dbc768e1608da5baab0a5896c54e840ff59110/docs/RULES.md#md009---trailing-spaces).